matrix_client/room.py

```diff
@@ -54,6 +56,32 @@ def set_user_profile(self,
             }
         )
 
+    @property
+    def display_name(self):
+        """
+        Calculates the display name for a room.
+        """
+        if self.name:
+            return self.name
+        elif self.canonical_alias:
+            return self.canonical_alias
+
+        members = self.get_joined_members()
+        # members without me
+        members = collections.OrderedDict({k: members[k] for k in members if
+                                           self.client.user_id != k})
+        first_two = list(members.values())[:2]
+        if len(first_two) == 1:
+            return first_two[0]["displayname"]
+        elif len(first_two) == 2:
+            return "{0} and {1}".format(
+                first_two[0]["displayname"],
+                first_two[1]["displayname"])
+        elif len(first_two) > 2:
+            return "{0} and {1} others".format(
+                first_two[0]["displayname"],
+                first_two[1]["displayname"])
```

Comment:
This should be the `len(members) - 1`

Reply:
uh yeah right fixed that ... but i have still a error for the room `#halium:matrix.org` its shown as `Halium` in riot but i only get the members list